app/services/idv/analytics_events_enhancer.rb

```diff
@@ -1,112 +1,91 @@
+      :idv_in_person_ready_to_verify_what_to_bring_link_clicked,
+      :idv_link_sent_capture_doc_polling_complete,
+      :idv_link_sent_capture_doc_polling_started,
+      :idv_native_camera_forced,
+      :idv_sdk_error_before_init,
+      :idv_sdk_selfie_image_capture_closed_without_photo,
+      :idv_sdk_selfie_image_capture_failed,
+      :idv_sdk_selfie_image_capture_initialized,
+      :idv_sdk_selfie_image_capture_opened,
+      :idv_sdk_selfie_image_re_taken,
+      :idv_sdk_selfie_image_taken,
+      :idv_selfie_image_added,
+      :idv_verify_in_person_troubleshooting_option_clicked,
+      :idv_warning_action_triggered,
+      :idv_warning_shown,
+    ].freeze
+
+    EXCLUDED_JOB_EVENT_METHODS = [
+      :idv_gpo_expired,
+      :idv_gpo_reminder_email_sent,
+      :idv_in_person_email_reminder_job_email_initiated,
+      :idv_in_person_email_reminder_job_exception,
+      :idv_in_person_proofing_enrollments_ready_for_status_check_job_completed,
+      :idv_in_person_proofing_enrollments_ready_for_status_check_job_ingestion_error,
+      :idv_in_person_proofing_enrollments_ready_for_status_check_job_started,
+      :idv_in_person_send_proofing_notification_attempted,
+      :idv_in_person_send_proofing_notification_job_completed,
+      :idv_in_person_send_proofing_notification_job_exception,
+      :idv_in_person_send_proofing_notification_job_skipped,
+      :idv_in_person_send_proofing_notification_job_started,
+      :idv_in_person_usps_proofing_enrollment_code_email_received,
+      :idv_in_person_usps_proofing_results_job_completed,
+      :idv_in_person_usps_proofing_results_job_deadline_passed_email_exception,
+      :idv_in_person_usps_proofing_results_job_deadline_passed_email_initiated,
+      :idv_in_person_usps_proofing_results_job_email_initiated,
+      :idv_in_person_usps_proofing_results_job_enrollment_incomplete,
+      :idv_in_person_usps_proofing_results_job_enrollment_updated,
+      :idv_in_person_usps_proofing_results_job_exception,
+      :idv_in_person_usps_proofing_results_job_please_call_email_initiated,
+      :idv_in_person_usps_proofing_results_job_started,
+      :idv_in_person_usps_proofing_results_job_unexpected_response,
+      :idv_in_person_usps_proofing_results_job_user_sent_to_fraud_review,
+      :idv_ipp_deactivated_for_never_visiting_post_office,
+      :idv_socure_reason_code_download,
+      :idv_socure_shadow_mode_proofing_result,
+      :idv_socure_shadow_mode_proofing_result_missing,
+      :idv_socure_verification_data_requested,
+      :idv_usps_auth_token_refresh_job_completed,
+      :idv_usps_auth_token_refresh_job_network_error,
+      :idv_usps_auth_token_refresh_job_started,
+    ].freeze
+
+    EXCLUDED_MISC_EVENT_METHODS = [
+      :idv_in_person_proofing_nontransliterable_characters_submitted,
+    ].freeze
+
+    IGNORED_METHODS = [
+      *EXCLUDED_FRONTEND_EVENT_METHODS,
+      *EXCLUDED_JOB_EVENT_METHODS,
+      *EXCLUDED_MISC_EVENT_METHODS,
+    ].uniq.freeze
```

Comment:
A/N: Using a `Set` causes some unexpected behaviors when doing collection comparisons as it behaves more like a `Hash` than an `Array`. Switched to using `uniq` since the const is immutable anyway.